CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/SingleChoiceInputPane.java

```diff
@@ -115,42 +115,42 @@ public SingleChoiceInputPane(final PluginParameter<SingleChoiceParameterValue> p
             SingleChoiceParameterType.setChoiceData(parameter, field.getSelectionModel().getSelectedItem());
         });
 
-        parameter.addListener((final PluginParameter<?> parameter1, final ParameterChange change) -> {
+        parameter.addListener((final PluginParameter<?> scParameter, final ParameterChange change) -> {
```

Comment:
Could be worth a check but potentially you may be able to replace `?` with `SingleChoiceParameterValue` here. If so, this would save the need to do the assert later on (and potentially even the type cast)

Reply:
When I tired it errors on compile :(

Reply:
ah never mind then